data/pycon-au-2011/videos/hosting-python-web-applications.json

```diff
@@ -1,29 +1,29 @@
 {
-  "id": 1002, 
-  "category": "PyCon AU 2011", 
-  "slug": "hosting-python-web-applications", 
-  "title": "Hosting Python Web Applications", 
-  "summary": "Web applications which are able to dynamically deliver up content\nhave&nbsp_place_holder;become a crucial part of the Internet landscape, with\nPython becoming an increasingly popular choice for the implementation of all\nmanner of web applications. In order though for users to be able to access\nyour web application you need to host it. The choices available are however\nmany and varied with accurate information on which may be a good choice not\nalways easy to find.\n\n", 
-  "description": "", 
-  "quality_notes": "", 
-  "language": "English", 
-  "copyright_text": "Creative Commons Attribution license", 
-  "thumbnail_url": "http://i.ytimg.com/vi/PWIvm-uloMg/hqdefault.jpg", 
-  "duration": null, 
+  "id": 1002,
+  "category": "PyCon AU 2011",
+  "slug": "hosting-python-web-applications",
+  "title": "Hosting Python Web Applications",
+  "summary": "Web applications which are able to dynamically deliver up content\nhave&nbsp\\_place\\_holder;become a crucial part of the Internet\nlandscape, with Python becoming an increasingly popular choice for the\nimplementation of all manner of web applications. In order though for\nusers to be able to access your web application you need to host it. The\nchoices available are however many and varied with accurate information\non which may be a good choice not always easy to find.\n",
```

Comment:
Another weird `have&nbsp\\_place\\_holder;`, but it was already [in the source page](http://pycon-au.org/2011/conference/schedule/event/27/index.html)

> to dynamically deliver up content haveÂ become a crucial part